Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "RigidBodyDynamics"
 uuid = "366cf18f-59d5-5db9-a4de-86a9f6786172"
-version = "2.3.2"
+version = "2.5.0"
```

Comment:
We could increment it to 2.4.0 instead.

Reply:
fixed, thanks

Reply:
Good to merge from my end (I don't have permissions to do it).